models/data/IPortfolio.ts

```diff
@@ -1,4 +1,36 @@
 /** Portfolio data */
+export interface IWorkHistory {
+  position?: string;
+  company?: string;
+  dateWorked?: string;
+  description?: string;
+}
+export interface IProjects {
+  title?: string;
+  picture?: string;
+  summary?: string;
+  viewCode?: string;
+  liveDemo?: string;
```

Comment:
Are there other kinds of links a person would want? For flexibility, this could be more generic like:


```
export interface ILink {
  label: string;
  url: string;
}

export interface IProjects {
  ...
  links: ILink[]
  ...
}
```

We'll be locked-into these API models forever, so we should be thoughtful about use cases